release.py

```diff
@@ -451,8 +451,7 @@ def generate_upload_tarball(args):
         tarball_name = "mentor2"
 
     # For ignition, we use the alias without version numbers as package name
-    if IGN_REPO:
-        tarball_name = re.sub(r'[0-9]+$','', args.package_alias)
+    tarball_name = re.sub(r'[0-9]+$', '', args.package_alias)
```

Comment:
I was trying to figure out why this line was limited to `IGN_REPO`; I'm guessing because of https://github.com/osrf/mentor2 just a few lines above? We can probably remove those lines at this point since that project is long over

at any rate, I think this line is safe

Reply:
I agree. I removed the mentor2/haptix references. a95ec0e